package.json

```diff
@@ -4,14 +4,16 @@
   "description": "Atomic Flux with hot reloading",
   "main": "lib/index.js",
   "scripts": {
-    "browser": "scripts/browser",
-    "build": "scripts/build",
-    "clean": "scripts/clean",
-    "lint": "scripts/lint",
-    "prepublish": "scripts/prepublish",
```

Comment:
Yes I think so. Could also skip linting. I think it's better to have granular “low-level” commands and make some higher level commands on top of them.